pkg/apis/core/types.go

```diff
@@ -3976,8 +3976,12 @@ type LocalObjectReference struct {
 
 // TypedLocalObjectReference contains enough information to let you locate the typed referenced object inside the same namespace.
 type TypedLocalObjectReference struct {
-	// APIGroup is the group for the resource being referenced
-	APIGroup string
+	// APIGroup is the group for the resource being referenced.
+	// If APIGroup is not specified, the specified Kind must be in the core API group.
+	// For any other third-party types, APIGroup is required. This is alpha feature gated
+	// and may change in the future.
```

Comment:
Does the alpha gate comment still apply? I think that can be dropped as well

Reply:
We still have the alpha feature gate for VolumeSnapshotDataSource.

Reply:
that's fine... but the alpha comment should go on the PersistentVolumeClaimSpec#DataSource field, not here

Reply:
Sure, will move over there.

Reply:
DataSource already has the following comments.  I'll just remove this feature gate comment here.

        // This field requires the VolumeSnapshotDataSource alpha feature gate to be
        // enabled and currently VolumeSnapshot is the only supported data source.